src/WooCommerce.php

```diff
@@ -672,4 +672,27 @@ public static function getProductAttributes(\WC_Product $product) {
     return $data;
   }
 
+  /**
+   * Adds custom query variable to filter products by delivery time.
+   *
+   * @implements query_vars
+   */
+  public static function query_vars($vars) {
```

Comment:
Orphan/obsolete remnants here to be removed, but otherwise it looks clean now!

Reply:
Crap! Sorry, cleaned up.